client/merchant-feedback-prompt/negative-modal.tsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useRef } from 'react';
+import { __ } from '@wordpress/i18n';
+import { Modal } from '@wordpress/components';
+import { dispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { recordEvent } from 'wcpay/tracks';
+import './style.scss';
+
+interface NegativeFeedbackModalProps {
+	onRequestClose: () => void;
+}
+
+export const NegativeFeedbackModal: React.FC< NegativeFeedbackModalProps > = ( {
+	onRequestClose,
+} ) => {
+	const textareaRef = useRef< HTMLTextAreaElement >( null );
+	// Record tracks event when the modal is opened.
+	useEffect( () => {
+		recordEvent( 'wcpay_merchant_feedback_prompt_negative_modal_view' );
+	}, [] );
+
+	return (
+		<Modal
+			title={ __( 'Share your feedback', 'woocommerce-payments' ) }
+			className="wcpay-merchant-feedback-negative-modal"
```

Comment:
I've added `negative` to this classname to distinguish it from the `positive` CSS tweaks in 2b19a5dfe4223c8d324ea5f693094a01cc5f8c44. Alternatively, we could combine the overridden styles, but this works as it is and is clear.

Reply:
Would `constructive` work? Slightly more positive framing 😁

Reply:
Yes, that could be an option. Since this is not user-facing, IMO `positive`/`negative` is a clear way to label and distinguish the two flows in code, "Yes" and "No".

This naming won't impact how we treat this feedback, e.g. I expect we'll see constructive criticism from the positive flow as well.